discordbot/OverworldDiscordBot.py

```diff
@@ -0,0 +1,282 @@
+# Needs a discord bot account and a server with all the mark ids as emojis.
+
+import asyncio
+import io
+import json
+import socket
+import sys
+from functools import lru_cache
+from threading import Thread
+from time import sleep
+from enum import Enum
+from distutils.util import strtobool
+
+import discord
+import pokebase as pb
+from discord.ext import commands
+from discord.utils import get
+from PIL import Image
+
+sys.path.append('../')
+from lookups import Util
+from nxreader import SWSHReader
+# from . import Channels
+
+class Channels(Enum):
+    NoChannel = 0
+    EventNotificationChannelIdForShiny = 1
+    EventNotificationChannelIdForBrilliant = 2
+    EventNotificationChannelIdForMark = 3
+    EventNotificationChannelIdForRareMark = 4
+    NotificationChannelForInfo = 5
+    TextNotificationChannelIdForShiny = 6
+    TextNotificationChannelIdForBrilliant = 7
+    TextNotificationChannelIdForMark = 8
+    TextNotificationChannelIdForRareMark = 9
```

Comment:
Note for future: to go with the filtering class below, channels would also need to be reworked to fit the custom filters

Reply:
Added as a future improvement in the readme.